plugins/jobs/web_client/views/JobListWidget.js

```diff
@@ -45,6 +69,17 @@ var JobListWidget = View.extend({
             obj[status.text] = true;
             return obj;
         }, {});
+        this.jobCheckedStates = {};
+        this.jobHighlightStates = {};
+        Object.defineProperty(this, 'jobs', {
```

Comment:
This is not directly related to Backbone, this is the es5 way to define property without ES6 class syntax.